lib/StandardResponse.php

```diff
@@ -205,15 +205,22 @@ public function flush() {
     /**
      * Signify the end of streaming response output.
      *
-     * User applications are NOT required to call Response::end() as the server
-     * will handle this automatically as needed.
+     * User applications are NOT required to call Response::end() after streaming
+     * or sending response data (though it's not incorrect to do so) -- the server
+     * will automatically call end() as needed.
      *
-     * Passing the optional $finalBody is equivalent to the following:
+     * Passing the optional $finalBodyChunk parameter is a shortcut equivalent to
+     * the following:
      *
-     *     $response->write($finalBody);
+     *     $response->write($finalBodyChunk);
      *     $response->end();
      *
-     * @param string $finalBody Optional final body data to send
+     * Note: Invoking Response::end() with a non-empty $finalBodyChunk parameter
+     * without having previously invoked Response::write() is equivalent to calling
+     * Response::send($finalBodyChunk).
```

Comment:
`Response::send()` doesn't exist.

Reply:
I've copied it from the interface, should probably be fixed there.

Reply:
Why did you copy it instead of using `@inheritdoc`?

Reply:
@bwoebi ping